### PR TITLE
wrap configurator checks in try/catch

### PR DESCRIPTION
**Pull request by grandvizier**

### CODE REVIEW

File: plugin/src/main/java/io/jenkins/plugins/casc/BaseConfigurator.java [hunk 1]
```diff
@@ -325,18 +325,26 @@ protected void configure(Mapping config, T instance, boolean dryrun, Configurati
...
-                                        configurator.configure(o, context);
-                        values.add(value);
+                        try{
+                            Object value =
+                                    dryrun ?
+                                            configurator.check(o, context):
+                                            configurator.configure(o, context);
+                            values.add(value);
+                        } catch (Exception ex) {
+                            throw new ConfiguratorException(configurator, "configuration is invalid", ex);
```
Comment (casz) [inline plugin/src/main/java/io/jenkins/plugins/casc/BaseConfigurator.java]: You should now throw but collect exceptions.

Comment (grandvizier) [reply plugin/src/main/java/io/jenkins/plugins/casc/BaseConfigurator.java]: yeah, I saw my solution wasn't the best one. But I'm not sure what you mean about collecting exceptions.

**Comment by ewelinawilkosz** [comment]
@grandvizier can you have a look at CI issues? (assuming you still want this PR to be merged)

**Comment by grandvizier** [comment]
you know, I worked on the code to create a for loop to catch the errors together, but in the end, I was no longer sure what the use case of this change was for, as in I wouldn't know what to test 

so I think i'm just going to close this PR

**Pull request closed without merging**